### PR TITLE
add gitee pulls plugin

### PR DESCRIPTION
**Pull request by kit101**

### CODE REVIEW

**Comment by d1wilko** [comment]
Hi,

We have recently updated the drone plugin index and docs are no longer written in markdown - they are now a more consistent yaml format e.g. [docker](https://github.com/drone/drone-plugin-index/blob/master/plugins/docker/content.yaml)

Please have a look at the new readme if you would like to run the new plugin index [readme](https://github.com/drone/drone-plugin-index/blob/master/README.md)

If you would still like to add documentation to the plugins index can you change the format to
- An individual folder per plugin inside the `plugins` folder
- a `content.yaml` file that follows the format in the example above - this is for quick discovery, not full documentation
- an `original.md` file that can contain the markdown you have already written
- a logo image inside the `public/logos` folder

Thanks :)

**Comment by kit101** [comment]
> Hi,
> 
> We have recently updated the drone plugin index and docs are no longer written in markdown - they are now a more consistent yaml format e.g. [docker](https://github.com/drone/drone-plugin-index/blob/master/plugins/docker/content.yaml)
> 
> Please have a look at the new readme if you would like to run the new plugin index [readme](https://github.com/drone/drone-plugin-index/blob/master/README.md)
> 
> If you would still like to add documentation to the plugins index can you change the format to
> 
> * An individual folder per plugin inside the `plugins` folder
> * a `content.yaml` file that follows the format in the example above - this is for quick discovery, not full documentation
> * an `original.md` file that can contain the markdown you have already written
> * a logo image inside the `public/logos` folder
> 
> Thanks :)

I will update these contents in these days. Thanks :)

**Comment by kit101** [comment]
Hi, @d1wilko . I have finished it.

File: plugins/gitee-pulls/content.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  - buildstatus
+  - teststatus
+logo: gitee.svg
+repo: https://github.com/kit101/drone-plugin-gitee-pulls
+image: https://hub.docker.com/r/kit101z/drone-plugin-gitee-pulls
+license: Apache License 2.0
+readme: https://github.com/kit101/drone-plugin-gitee-pulls/blob/master/README.md
+description: This plugin can automatically update the comments / labels / test in PR to gitee.
+example: |
+  ---
```
Comment (d1wilko) [inline plugins/gitee-pulls/content.yaml:15]: can you remove this line?

File: plugins/gitee-pulls/content.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  name: default
+  kind: pipeline
+  type: docker
+
+  #label has default values
+  #environment:
+  #  PLUGIN_GITEE_RUNNING_LABEL: drone-build/running,E6A23C
+  #  PLUGIN_GITEE_SUCCESS_LABEL: drone-build/success,67C23A
+  #  PLUGIN_GITEE_FAILURE_LABEL: drone-build/failure,DB2828
```
Comment (d1wilko) [inline plugins/gitee-pulls/content.yaml:24]: can you remove these lines?

File: plugins/gitee-pulls/content.yaml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+  steps:
+  - name: pr-enhance/start
+    pull: always
+    image: kit101z/drone-plugin-gitee-pulls
+    settings:
+      # should set `is_running: true` in the first step
```
Comment (d1wilko) [inline plugins/gitee-pulls/content.yaml:31]: can you remove this line?

File: plugins/gitee-pulls/content.yaml [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    image: kit101z/drone-plugin-gitee-pulls
+    settings:
+      access_token:
+        from_secret: GITEE_ACCESS_TOKEN
+    when:
+      event:
+      - pull_request
+      status:
+      - failure
+      - success
```
Comment (d1wilko) [inline plugins/gitee-pulls/content.yaml:54]: can you remove these lines?

**Review by d1wilko — CHANGES_REQUESTED** [review]
Thank you very much for updating, it looks great.

Can I ask you to remove some lines from the example - we want to keep it extremely simple with more complicated examples in the markdown and repo readme :)

**Comment by kit101** [comment]
> Thank you very much for updating, it looks great.
> 
> Can I ask you to remove some lines from the example - we want to keep it extremely simple with more complicated examples in the markdown and repo readme :)

I couldn't agree more. I've removed these code. :)

**Review by d1wilko — APPROVED** [review]
Looks good to me :) - thank you

**Pull request merged**